api/qanpb/collector.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package qan;
+package qan.v1beta1;
```

Comment:
why did you change it?

Reply:
https://github.com/uber/prototool/blob/dev/docs/breaking.md#beta-vs-stable-packages